app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -127,12 +125,6 @@ protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            Window w = getWindow();
-            w.setFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS,
-                    WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
-        }
-
```

Comment:
Since you removed this code, according to https://developer.android.com/reference/android/view/WindowManager.LayoutParams#FLAG_TRANSLUCENT_STATUS you should use a half-translucent color for the toolbar, to get the same result as before.

Reply:
### Clarifications:
1. Do we want the status bar to look the same as it did previously? I interpreted #3013 as looking for the same color status and toolbar instead of having the transparent dark overlay.
2. If we change the alpha of the status bar color it will blend with the background colors \#EEEEEE for the light theme, \#222222 for the dark theme, and \#000000 for the black theme which causes it to look even further from the toolbar color. In this case we could balance the color so that the change of alpha does not affect visual appearance, however, this requires balancing for every background color and service color option (it is also not very straightforward.)
3.	Lastly, looking at Google’s own apps that use the matching status/toolbar colors (e.g. clock, hangouts, and photos) their status bars appear to have alpha values of one.

### Google Apps:

![Google Photos status bar color](https://user-images.githubusercontent.com/56372040/84603394-fe8dc580-ae42-11ea-9582-c5a0c0c8dbb1.png)

![Google Clock status bar color](https://user-images.githubusercontent.com/56372040/84603395-fe8dc580-ae42-11ea-92d9-3c7d7a9f4703.png)

### Without Matching: 

![Newpipe status bar color with half alpha and side menu open](https://user-images.githubusercontent.com/56372040/84603389-f6ce2100-ae42-11ea-9f0b-60934dd5f83e.png)

![Newpipe status bar color with half alpha on trending tab](https://user-images.githubusercontent.com/56372040/84603390-f766b780-ae42-11ea-8b23-0204a56227f0.png)

### Rough Match:

![Newpipe status bar color with half alpha, attempted match, and side menu open](https://user-images.githubusercontent.com/56372040/84603365-c71f1900-ae42-11ea-94d0-b992df4d88cc.png)

![Newpipe status bar color with half alpha and attempted match on trending tab](https://user-images.githubusercontent.com/56372040/84603376-d3a37180-ae42-11ea-87c9-a90b5b47d322.png)